rpm/valgrind.spec

```diff
@@ -27,6 +27,13 @@ malloc/new/free/delete are intercepted. As a result, Valgrind can
 detect a lot of problems that are otherwise very hard to
 find/diagnose.
 
+%package extratools
```

Comment:
Zypper at least requires user confirmation before installing the dependencies. Maybe could add some note on package summary and description, e.g. "(requires Perl)"?